src/zh/manwa/src/eu/kanade/tachiyomi/extension/zh/manwa/Manwa.kt

```diff
@@ -176,26 +175,25 @@ class Manwa : ParsedHttpSource(), ConfigurableSource {
     override fun imageUrlParse(document: Document): String = throw UnsupportedOperationException()
 
     override fun setupPreferenceScreen(screen: PreferenceScreen) {
-        ListPreference(screen.context).apply {
+        EditTextPreference(screen.context).apply {
```

Comment:
please implement in [such a way](https://github.com/keiyoushi/extensions-source/blob/71d5f45eacbf5e41db821f08af1df638ecd8c5a6/src/ar/teamx/src/eu/kanade/tachiyomi/extension/ar/teamx/TeamX.kt#L230-L261)